source/index.html

```diff
@@ -48,43 +54,45 @@ <h2>ЕДА</h2>
             </li>
           </ul>
         </li>
-        <li class="advantages__item">
+        <li class="advantages__item advantages__block--blue">
           <h1 class="advantages__title">ТАМ ЕСТЬ МОСТ ДЬЯВОЛА</h1>
-          <p class="advantages__description">№2</p>
+          <p class="advantages__description">— №2 —</p>
           <p class="advantages__description">ДА, ПО НЕМУ МОЖНО ПРОЙТИСЬ! ЕСЛИ ВЫ ОСМЕЛИТЕСЬ, КОНЕЧНО</p>
-          <img src="https://via.placeholder.com/320x260">
-        </li>
-        <li class="advantages__item">
-          <h1 class="advantages__title">НЕБОЛЬШАЯ ПЛОЩАДЬ</h1>
-          <p class="advantages__description">№3</p>
-          <p class="advantages__description">ВСЕ ИНТЕРЕСНЫЕ МЕСТА НАХОДЯТСЯ ОЧЕНЬ БЛИЗКО</p>
-        </li>
-        <li class="advantages__item">
-          <h1 class="advantages__title">КРАСИВАЯ ДОРОГА</h1>
-          <p class="advantages__description">№4</p>
-          <p class="advantages__description">ЕХАТЬ В СЕДОНУ ИЗ ЛАС-ВЕГАСА СОВСЕМ НЕ СКУЧНО!</p>
-        </li>
-        <li class="advantages__item">
-          <h1 class="advantages__title">МАЛО ТУРИСТОВ</h1>
-          <p class="advantages__description">№5</p>
-          <p class="advantages__description">БОЛЬШИНСТВО ЕДЕТ В ГРАНД КАНЬОН И ТОЛПИТСЯ ТАМ</p>
+          <img src="img/main-bridge-mobile@1x.jpg" width="320">
         </li>
+        <ul class="advantages__item advantages__item--gray">
+          <li class="advantages__item advantages--devil">
```

Comment:
Первое, это модификатор написан неправильно, должно быть так:
```suggestion
          <li class="advantages__item advantages__item--devil">
```

Reply:
второе, это для списка и его элемента испольуется один и тот же класс - `advantages__item`. Так быть не должно